### PR TITLE
BUG: Completes ConsumeAssetMetaDataError message

### PR DESCRIPTION
**Pull request by jfkirk**
@ssanderson Is this message satisfactory?

### CODE REVIEW

File: zipline/errors.py [hunk 1]
```diff
@@ -259,7 +259,9 @@ class ConsumeAssetMetaDataError(ZiplineError):
     Raised when AssetMetaData.consume() is called on an invalid object.
     """
     msg = """
-AssetMetaData can not consume {obj}. MetaData must be a dict, a DataFrame, or
+AssetFinder can not consume metadata of type {obj}. Metadata must be a dict, a
+DataFrame, or an object with a 'read' attribute. If the provided metadata has a
```
Comment (ssanderson) [inline zipline/errors.py]: Lots of types have `read` methods (notably, any open file object).  I would refer to this specifically as a `tables.Table`, which is what we actually support.

File: zipline/errors.py [hunk 2]
```diff
@@ -259,7 +259,9 @@ class ConsumeAssetMetaDataError(ZiplineError):
     Raised when AssetMetaData.consume() is called on an invalid object.
     """
     msg = """
-AssetMetaData can not consume {obj}. MetaData must be a dict, a DataFrame, or
+AssetFinder can not consume metadata of type {obj}. Metadata must be a dict, a
+DataFrame, or a tables.Table. If the provided metadata is a Table, the rows
```
Comment (jfkirk) [inline zipline/errors.py]: @ssanderson tables.Table is now explicit

**Comment by ssanderson** [comment]
@jfkirk LGTM

**Comment by jfkirk** [comment]
@ssanderson Thanks for merging this!

**Pull request merged**